webpack/components/QuestionList.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component, PropTypes } from 'react';
+var QuestionWidget = require('./QuestionWidget.js').default;
```

Comment:
Let's not mix import and require.
Change to:
`import QuestionWidget from './QuestionWidget';`